<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 4.1.3 (Python 3.9.10 on linux)" generated="20220314 15:06:56.761" rpa="false" schemaversion="2">
<suite id="s1" name="Demo1 Orangehrm Suite" source="/home/Indranee123/home/Indranee123/Documents/Robotframework/Robot_project/selenium_suite/demo1_orangehrm_suite.robot">
<test id="s1-t1" name="TC1">
<kw name="Append To Environment Variable" library="OperatingSystem">
<arg>Path</arg>
<arg>${EXECDIR}${/}driver${/}chromedriver</arg>
<doc>Appends given ``values`` to environment variable ``name``.</doc>
<msg timestamp="20220314 15:06:56.783" level="INFO">Environment variable 'Path' set to value '/home/Indranee123/home/Indranee123/Documents/Robotframework/Robot_project/selenium_suite/driver/chromedriver'.</msg>
<status status="PASS" starttime="20220314 15:06:56.783" endtime="20220314 15:06:56.783"/>
</kw>
<kw name="Log To Console" library="BuiltIn">
<arg>${EXECDIR}</arg>
<doc>Logs the given message to the console.</doc>
<status status="PASS" starttime="20220314 15:06:56.783" endtime="20220314 15:06:56.783"/>
</kw>
<kw name="Open Browser">
<arg>browser=chrome</arg>
<msg timestamp="20220314 15:06:56.785" level="FAIL">No keyword with name 'Open Browser' found.</msg>
<status status="FAIL" starttime="20220314 15:06:56.784" endtime="20220314 15:06:56.785"/>
</kw>
<status status="FAIL" starttime="20220314 15:06:56.782" endtime="20220314 15:06:56.785">No keyword with name 'Open Browser' found.</status>
</test>
<status status="FAIL" starttime="20220314 15:06:56.761" endtime="20220314 15:06:56.785"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Demo1 Orangehrm Suite">Demo1 Orangehrm Suite</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20220314 15:06:56.782" level="ERROR">Error in file '/home/Indranee123/home/Indranee123/Documents/Robotframework/Robot_project/selenium_suite/demo1_orangehrm_suite.robot' on line 4: Importing library 'SeleniumLibrary' failed: ModuleNotFoundError: No module named 'SeleniumLibrary'
Traceback (most recent call last):
  None
PYTHONPATH:
  /usr/local/bin
  /usr/lib64/python39.zip
  /usr/lib64/python3.9
  /usr/lib64/python3.9/lib-dynload
  /usr/local/lib/python3.9/site-packages
  /usr/lib64/python3.9/site-packages
  /usr/lib/python3.9/site-packages</msg>
</errors>
</robot>
